docs/options.md

```diff
@@ -53,8 +53,10 @@ new SVGSpritemapPlugin(string | string[], {
     },
     styles?: boolean | string | {
         filename?: string,
-        format?: 'data' | 'fragment',
+        format?: 'data' | 'fragment' | 'dimensions',
         keepAttributes?: boolean,
+        includeDimensions?: boolean,
+        units?: 'rem' | 'px',
```

Comment:
I'm not sure having 3 separate options that all converge into the same feature makes sense for end-users. What do you think about the following setup:

* `format` is able to be set to `false` which will remove the `background-image: url()` part from the output
* `includeDimensions` and `units` are replaced with a single `dimensions` option, which can be set...
  * To `false` (default) to not include `width`/`height` (i.e. what's available in the current release)
  * To `'rem'` or `'px'` to have `width`/`height` included with the specified unit.

This gets rid of any illogical combinations of options that will just cause confusion, like setting `format` to `data` and then setting `units` to `'px'`.